cmd/cmd-mewn.go

```diff
@@ -5,6 +5,6 @@ package cmd
1af2ab3d3a3595a8f228137e44c5477b1b8846a08e8e578f32e9b20bd6878c32996af5578c42067b86bf6624f89cd5e3b1d85b3cf6568fbfc5e39284640d3b1c3434776d4b8c206b021eba40dcc0f495a11d0357877f45470a313e28a3515265cc37ff1abb084cb0d025d26c756501e569d505cca165825825138c6f4d4808d60230cd80cb9cbb64b6e2904cf0ad03897394109d9aa432e6c391394a06b002756400f0e0aedf6da655c6c6b0e1e812a98cd5e3b1370426832328761ca09a6210534bbe134fe0f56c0d7c4d04d3b7401903b01655c65f7fff7edcd08187aab1c6768cf51eb62099106089b37f67fe538fc7c4f3096b21cbd9dee48acf9e218ca8ad012214710350d3b68e660e24f4a50be2f5dcb731503ede3a708e88761ecc1082f7390afbdfc37d689f5f4684ccded0440b6caba58940245a600b7174e0b187386c088bef9f23f01141043d91b5c43426a083a3003650eaf47684382ef0954b91b637d50838432bc7375098138f4662c0574e38a5260fbacf6c6d1b555facebc1911ff3c13c264c2b843ada59c09da38446d6c21b83841c8ea942b277b827aafd384ba72393683deecfdf1d3f72a29a2065348ac78e11f5391965343a78cf88633720139d2f235611e7c1e661f61f094b9d1831766a3c21e307ec0f4337baf467ecfa8750d1b7023f057cff6a217a4dcb15b8db1198a072db1569fad3313411f7b3d8e782ec5f25a04b719d2cdc052d11dec4c8b03d766a8da3f50cdb1bbe133330e3ef281464a67438c30afe33d3565f13bed3fbbcaf89069ff1368806ff9fbc457b10612d6b9ac8394c7589c311b7f75927ed9f0ec1b2d7c010e188cc1113c435e46051ad39b88830c4dd15a423e2624d1bf9b49201c973e4fa34028945bcb19fd5041fc70fad18fb594958e0353e0516db4a904dd065a88f3839c9d827b985f90e7e5413f96d14c5e6bee4b3a0fef916e977e3313f9945d589b17490b3fce00f4e8eb483921ade30a69a290c4955db225447c7203afa278b5ebb8fa004679fdfb7e85e12cade97c5fd856f7502b37cec3ce2443501bb1da1f01a9b1f259968fcfa183fcb04699baaa6cd44700582370f11f61ba8fb0e6b8fdc0dd69f607e10cfd1607e6ff1a3a18f51ffa3a18ff2fec2d6cd48ea08dce3c4e0bf972eff6f692a16fb380fd58fef4b936747038cddadf087f0641fdaa1b408d6e06e513e93b1184a6a30c72b93a3353657701684be6260aa03e20661c1febe25b130a128fa863878791c4d02bf34e5bbe2c0801f5cf1d8ce87312682eee7660dec1a82fff20ac0438f301bf9eb0f09b3c9f0ce37ea2f1dd776801ebc5639fafb2f11c1756da1f851d97d4689c520cdb8ed742c0b348239dcbbb683757fc64a865285201f9f38ae3f0736832b3d1cfca663c662fb6e6b8e4d06c1d69bffe74d24b4f38fda9ad3af00f62ef82ec2c2a03e1dd2440112e527f4240f10d1a9ed025a3084a926daf770b808d2638c7984abc661d44b6b98e37f67fe73a016743f2699b9d8dda28b10f95ffef7020c50c24bf86fb38f552a4a28cd4ca242327b16140641af677be150e8ae282716889a27c654d0cec383b9edfa71e102d6b6c8377e88ec6b107e1db18f3f3f7ab0e3c4dfdd49bae54fefe93e4d066d662de09238473f44c4a7c929d07e2804498417ff49f4ecc53a78dd67247b3627f3ada858d8714c127c7993f88957ed299f700de1df7c3f0a169f31bcf3d50e2f7218183de4377301bfd60f37c3c93c94871dee97fed6fd9efb6dcb07441b848a61f9406e1690f32788e0085da20f498614840a5f422a35f405a5f678673f5cf84bba075e089adc43fd2892824dd3cfda07a1d4a3507b633731c73f95ec1d67657f495102ef988745a1f603fa9dada75b2281251313fa4a6d8f1ede91881e0db9a0025983568bbe3e3944ea695f14cc2db3addf620bf3469ae0743a1dffb4e4f9d8e9745ac418e61caca07b0d01e3d182f8167f370725a3debf1f85f60365e08b620223fcc06884d7e09fee313f9e34f6b17e7980ab47d50c87b2d8c930762a5f47703d09ce7712fbe096adad4c48c3c6b15d2efa6981568fff6f000000ffffc3c33335cf1e0000")
```

Comment:
Why is this file marked as changed?

Reply:
I've honestly no clue. I think has something to do with changing branch, but the content looks the same. I can't see why it's being highlighted

Reply:
Ah, that sucks, but not a deal breaker :-)